components/ExerciseDetails/index.js

```diff
@@ -2,40 +2,108 @@ import styled from "styled-components";
 
 export default function ExerciseDetails({ exercise }) {
   return (
-    <>
-      <figure>
+    <Container>
+      <ImageWrapper>
         <ExerciseImage src={exercise.imageUrl} />
+      </ImageWrapper>
+      <TitleAndMuscleWrapper>
+        <ExerciseTitle>{exercise.name}</ExerciseTitle>
         <ExerciseDetailsMuscle>
-          {exercise.muscleGroups.join(" ")}
+          {exercise.muscleGroups.map((muscle, index) => (
+            <MuscleGroup key={index}>{muscle}</MuscleGroup>
+          ))}
         </ExerciseDetailsMuscle>
-      </figure>
+      </TitleAndMuscleWrapper>
       <ExerciseDetailsInstruction>
         {exercise.instructions.map((instruction, index) => (
-          <li key={index}>{instruction}</li>
+          <InstructionItem key={index}>
+            <InstructionNumber>{index + 1}</InstructionNumber>
+            {instruction}
+          </InstructionItem>
         ))}
       </ExerciseDetailsInstruction>
-    </>
+    </Container>
   );
 }
 
+const Container = styled.div`
+  font-family: Arial, sans-serif;
```

Comment:
Since your userstory is about global styling you should use font variables instead.